tests/libtest/Makefile.inc

```diff
@@ -36,7 +36,7 @@ SUPPORTFILES = first.c test.h
 
 # These are all libcurl test programs
 noinst_PROGRAMS = chkhostname libauthretry libntlmconnect                \
- chkdecimalpoint                                                         \
+ chkdecimalpoint libprereq                                               \
```

Comment:
Why doesn't this follow the pattern of being named `lib[num]` ?

Reply:
Because it's generic over a number of different tests. AFAICT this is the pattern for that (see libntlmconnect).